metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -852,6 +852,12 @@ class MetalsLanguageServer(
                   s"${config.icons.check}Compiled$name in $elapsed"
                 )
               case StatusCode.ERROR =>
+                statusBar.addMessage(
+                  MetalsStatusParams(
+                    s"${config.icons.alert}Compile error in $elapsed",
```

Comment:
Nitpick: "Compile error in x" sounds a lot like a countdown. Maybe some parenthesis could help?

Reply:
It's a good point, what about?

```
Compiled foo (1.2s)
Error in foo (1.2s)
```

The two should be consistent

Reply:
If those two come with optional icons, I'm sold 👌

Reply:
Realized `Error in foo` might be misleading since the compile error can be in a dependency project.

```
✅ Compiled foo (1.2s)
⚠️ Compile error (1.2s)
```

The icons will be there :+1: